src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -318,17 +318,19 @@
 					{{ t('spreed', 'Also ban from this conversation') }}
 				</NcCheckboxRadioSwitch>
 				<template v-if="isBanParticipant">
-					<NcTextField v-if="isBanParticipant"
+					<NcTextArea v-if="isBanParticipant"
 						class="participant-dialog__input"
 						:label="t('spreed', 'Internal note (reason to ban)')"
+						:error="!!maxLengthWarning"
+						:helper-text="maxLengthWarning"
 						:value.sync="internalNote" />
```

Comment:
This to be fixed in upstream (specifying max width and max height) ?
![image](https://github.com/user-attachments/assets/38c21f37-4cb5-4e88-913b-09207f72ca8f)

Reply:
`resize="vertical"` is enough to prevent overflow